modules/era/sql/era_droplist.sql

```diff
@@ -12424,15 +12421,15 @@ INSERT INTO `mob_droplist` VALUES (1502,0,0,1000,2532,20);        -- Teak Log (2
 INSERT INTO `mob_droplist` VALUES (1503,0,0,1000,18445,@VCOMMON); -- Amakura (Very Common, 24%)
 
 -- ZoneID: 107 - Leaping Lizzy
-INSERT INTO `mob_droplist` VALUES (1504,0,0,1000,852,@COMMON);   -- Lizard Skin (Common, 15%)
 INSERT INTO `mob_droplist` VALUES (1504,0,0,1000,926,@VCOMMON);  -- Lizard Tail (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (1504,0,0,1000,13014,@RARE);   -- Leaaping Boots (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1504,0,0,1000,13014,@COMMON); -- Leaaping Boots (Common, 15%)
```

Comment:
![image](https://user-images.githubusercontent.com/105882754/183808884-0818459c-ac7f-4ef7-b0fa-5ef9927dbc73.png)

I would use this drop rate @5% instead of the FFXIDB. It has over 3k data points instead of ffxidbs 600